tests/tracking/test_rest_tracking.py

```diff
@@ -51,9 +52,14 @@ def _await_server_down_or_die(process, timeout=60):
     """Waits until the local flask server process is terminated."""
     print('Awaiting termination of server process...')
     start_time = time.time()
-    while process.is_alive() and time.time() - start_time < timeout:
+
+    def wait():
```

Comment:
why was this change required? is it more platform independent?

Reply:
The multiprocessing on windows is weird. I did not find a way how to start a Flask server using the Process abstraction. I had to use Popen instead so the process object is different. unfortunately python2 does not have a timeout argument in wait() and I did not find other way of fetching the return code other then calling wait(). And since we want to avoid a hang, the wait is called from a new thread.